src/js/themes/hpe.js

```diff
@@ -1259,6 +1259,15 @@ const buildTheme = (tokens, flags) => {
       container: { cssGap: true, gap: 'small', margin: 'none' },
     },
     data: { button: { kind: 'toolbar' } },
+    dataFilter: {
+      rangeSelector: {
+        size: 'full',
+        round: 'xsmall',
```

Comment:
```suggestion
        round: 'medium',
```
I think "medium" is correct looking at the values here for radius? https://github.com/grommet/hpe-design-system/issues/4487#issuecomment-2576435972

Reply:
was "small", so now should be "medium"?